### ISSUE TITLE
Parse error when output.xml contains latin characters

### ISSUE THREAD

**Issue by xNok** [adiralashiva8/robotframework-metrics#76]
It seems that the code is not encoding free (Given #41)

```
table_td.string = str(kw.kwname)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe0' in position 36: ordinal not in range(128)
```

On solution may be to sanitise the xml input to make robotframework-metrics error free. It is frustrating that the script does not go throught for 1 unrecognised character. However it would not matter if that character was replace by ?.

```
u.encode('ascii', 'replace')
```

**Comment by adiralashiva8** [comment]
@xNok In recent change I have modified `robotmetrics` to use `uft-8` decode, could you please try reported issue with latest change

Change - https://github.com/adiralashiva8/robotframework-metrics/blob/a4c0c60bd2d2e40c1b4f33f67cb13f2d7c441436/setup.py#L4

For latest (master) changes installation use `pip install git+https://github.com/adiralashiva8/robotframework-metrics`

**Comment by xNok** [comment]
This is solved as of v3.1.4, Howerver v3.1.4 as not been release to pypi so impossible to get it from `pip install`

**Comment by adiralashiva8** [comment]
Use following command to install latest changes until new release!

pip install git+https://github.com/adiralashiva8/robotframework-metrics
